bootstrap/bootstrap_storage.sh

```diff
@@ -35,10 +35,15 @@ else
 fi
 
 TABLE_INITIALIZER_MAIN_CLASS=org.openmetadata.catalog.util.TablesInitializer
-for file in "${BOOTSTRAP_DIR}"/../libs/*.jar;
-do
-    CLASSPATH="$CLASSPATH":"$file"
-done
+LIBS_DIR="${BOOTSTRAP_DIR}"/../libs/
```

Comment:
What's the reason behind this. Lets not mix changes for one PR into another, if its needed lets open separate PR

Reply:
Ok